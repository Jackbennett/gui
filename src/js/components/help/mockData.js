export const helpProps = {
  docsVersion: '',
  findLocalIpAddress: jest.fn,
  isHosted: false,
  isEnterprise: false,
  menderDebPackageVersion: 'master',
  menderVersion: 'master',
  menderArtifactVersion: 'master',
  org: {
    tenant_token: ''
  }
};
