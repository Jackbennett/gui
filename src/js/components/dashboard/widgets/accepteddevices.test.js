import React from 'react';
import renderer from 'react-test-renderer';
import AcceptedDevices from './accepteddevices';
import { undefineds } from '../../../../../tests/mockData';

describe('AcceptedDevices Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<AcceptedDevices />).toJSON();
    expect(tree).toMatchSnapshot();
    expect(JSON.stringify(tree)).toEqual(expect.not.stringMatching(undefineds));
  });
});
