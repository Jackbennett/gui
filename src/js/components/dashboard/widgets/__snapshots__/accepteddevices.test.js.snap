// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AcceptedDevices Component renders correctly 1`] = `
<div
  class="notActive widget "
>
  <div
    style="height: 100%; width: 100%; display: flex; flex-direction: column;"
  >
    <div
      class="widgetMainContent align-right"
      style="display: flex; flex-direction: column; align-items: flex-end;"
    >
      <div
        class="header"
      >
        Accepted devices
      </div>
      <div
        class="counter"
      />
    </div>
    <span
      class="link"
    />
  </div>
</div>
`;
